packages/nimble-components/src/anchor/specs/README.md

```diff
@@ -0,0 +1,111 @@
+# Nimble Anchor
+
+## Overview
+
+The Nimble Anchor is a component used to navigate to a web resource, similar to the HTML anchor element.
+
+### Background
+
+[Visual Design spec](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/bfadf499-caf5-4ca0-9814-e777fbae0d46)<br>
+[Nimble work item](https://github.com/ni/nimble/issues/324)
+
+---
+
+## Design
+
+### API
+
+[FAST's API documentation](https://github.com/microsoft/fast/blob/e576aa70c22780fffba03097277e2db9a2ec1cd8/packages/web-components/fast-foundation/src/anchor/README.md)
+
+-   _Component Name_: `nimble-anchor`
+-   _Properties/Attributes_: We will have the following properties/attributes in addition to the ones provided by the FAST anchor:
+    -   `appearance`:
+        -   `"text"`: (Default) the "standalone" version in the design doc. Looks like plain text, but gets and underline on hover.
+        -   `"inline-text"`: like `"text"`, but always shows an underline.
+        -   `"outline"`: same as button design
+        -   `"ghost"`: same as button design
+        -   `"block"`: same as button design
+    -   `appearance-variant`:
+        -   `"default"`: `undefined` (as per our common attribute guidelines)
+        -   `"primary"`: applies only to button-based styles and has the same effect as on buttons
+        -   `"prominent"`: applies only to `"text"` and `"inline-text"` appearances. It is the "loud" version from the design doc. Colors the link text green. The name of this attribute value comes from the Breadcrumb.
+    -   `content-hidden`: when set, hides the label and end slot
+    -   `disabled`: when set, makes the anchor inoperable and changes the styling
+-   _Methods_: Unchanged (none)
+-   _Events_: Unchanged (none)
+-   _CSS Classes and Custom Properties that affect the component_: Unchanged (none)
+-   _Slots_: Unchanged (start and end slots)
+
+We are not implementing navigation tabs as part of this effort. That might be implemented in the future by populating a `nimble-tabs` control with `nimble-anchor`s. It may require a new appearance mode.
+
+### Angular integration
+
+A directive will be created for the anchor component. It will extend the `NimbleBreadcrumbItemDirective` since it has all the same bindings for attributes common to the native anchor element (e.g. `href`, `rel`, `target`, etc.). It will additionally have bindings for attributes `appearance`, `appearance-variant`, `content-hidden`, and `disabled`.
```

Comment:
> A directive will be created for the anchor component. It will extend the `NimbleBreadcrumbItemDirective`

If there is an inheritance relationship between the controls does that sound like the right one? All links are breadcrumb items?

Reply:
No, that's a good point. I've modified the spec to say that we will refactor the common bindings into a common base class rather than the anchor inheriting directly from the breadcrumb item.